limesurvey/templates/deployment.yaml

```diff
@@ -197,6 +202,14 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
+      {{- if .Values.ldapConfigMap }}
+        - name: ldap-config
+          configMap:
+            name: {{ include "limesurvey.ldapConfigMapName" . }}
+            items:
+            - key: "ldap.php"
```

Comment:
Not sure if I understand the docs correctly, but won't this mount the ldap.php to `keys/ldap.php`?
https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/volume/#KeyToPath

Reply:
Think I misunderstood that. I just pushed some code that I tested.